README.md

```diff
@@ -1,59 +1,48 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Portfolio
 
```

Comment:
- [ ] **OPTIONAL:** You have set up PR correctly and provides short summary and descriptive title, but I strongly recommend you to use links with markdown and add steps to what you have done. 

_I attached from where you can add links, and an example of descriptive summary_

![image](https://user-images.githubusercontent.com/77872355/129307264-2ca5854e-cbeb-4d51-9257-78e6644928b0.png)

![image](https://user-images.githubusercontent.com/77872355/129307309-0f78c1e6-39f5-479d-8197-fe4a9233c87d.png)